atlasdb-cassandra/src/test/java/com/palantir/atlasdb/cassandra/CassandraReloadableKeyValueServiceRuntimeConfigTest.java

```diff
@@ -79,11 +80,10 @@ public void mutationBatchCountAlwaysUsesRuntimeConfigValue() {
 
     @Test
     public void mutationBatchSizeBytesAlwaysUsesRuntimeConfigValue() {
```

Comment:
you might be interested to see where `mutationBatchSizeBytes` is actually used; I don't think we're actually merging this appropriately.

Reply:
Addressed - all those changes are in isolation to make it slightly easier to review, even though all install -> runtime changes happen in one go: https://github.com/palantir/atlasdb/pull/6050/commits/bccc7703530ebbf78ff3e679aeab40c3487b3d74

Reply:
One thing to note: The merging was rather weird - for these parameters, iff the runtime config was present, then the runtime config value was used. In most cases, the runtime config isn't present (even alta didn't have one for cassandra), so we'd actually fall back to the install values. 

Comparatively, servers explicitly used install if present even if the runtime config was present.

Looking at our internal search tools, I believe it's safe to change how the merging worked - use install if override present, then use runtime. This is almost entirely the same behaviour in practice, given that the runtime config block was rarely specified.